components/server/src/user/user-deletion-service.ts

```diff
@@ -132,6 +135,11 @@ export class UserDeletionService {
         }
     }
 
+    protected async deleteTeamMemberships(userId: string) {
+        const teams = await this.teamDb.findTeamsByUser(userId);
+        await Promise.all(teams.map(t => this.teamDb.removeMemberFromTeam(userId, t.id)));
```

Comment:
👍🏻

Reply:
@AlexTugarev Did you mean to `/lgtm` this Pull Request? 😇